src/features/content-insights/ContentAnalyticsErrorState.tsx

```diff
@@ -1,18 +1,59 @@
 import * as React from 'react';
 import { FormattedMessage } from 'react-intl';
 
-import HatWand140 from '../../illustration/HatWand140';
+import PropTypes from 'prop-types';
+import MetricsReview56 from '../../illustration/MetricsReview56';
 import messages from './messages';
+import { ResponseError } from './types';
 
 import './ContentAnalyticsErrorState.scss';
 
-const ContentAnalyticsErrorState = () => (
-    <div className="ContentAnalyticsErrorState" data-testid="ContentAnalyticsErrorState">
-        <HatWand140 data-testid="ContentAnalyticsErrorState-image" />
-        <div className="ContentAnalyticsErrorState-text">
-            <FormattedMessage {...messages.contentAnalyticsErrorText} />
+const SIZE_MAP = {
```

Comment:
Instead of javascript, can we apply the styles via CSS in each use case? i.e. in `ContentInsightsSummary.scss` style it to be the small size, and in the flyout and modal, style it as appropriate